src/UI/templates/default/Popover/tpl.popover.html

```diff
@@ -0,0 +1 @@
+<div class="webui-popover il-popover" role="tooltip"><div class="webui-arrow"></div><div class="webui-popover-inner"><a href="#" class="close"></a><h3 class="webui-popover-title"></h3><div class="webui-popover-content"><i class="icon-refresh"></i><p>&nbsp;</p></div></div></div>{FORCE_RENDERING}
```

Comment:
If I remember correctly,`JSON.parse()` was not happy if the template contains linebreaks. I'm not sure why PHP's `json_encode` doesn't take care of that, but I will try to escape it myself